tests/conftest.py

```diff
@@ -78,10 +80,31 @@ def local_http_server():
 
 
 @pytest.fixture
-def http_schema_file_url(local_http_server):
-    "The base of remote http schema file URLs."
+def schema_file_absolute_dir() -> Path:
+    "The base of the absolute path to the dir that contains test schema files."
 
-    return f"http://{local_http_server.server_address[0]}:{local_http_server.server_address[1]}/tests/schemata/"
+    return Path.cwd() / 'tests' / 'schemata'
+
+
+@pytest.fixture
+def schema_file_relative_dir() -> Path:
+    "The base of the relative path to the dir that contains test schema files."
+
+    return Path('tests/schemata')
+
+
+@pytest.fixture
+def schema_file_file_url(schema_file_absolute_dir) -> str:
```

Comment:
```suggestion
def schema_file_file_uri(schema_file_absolute_dir) -> str:
```
URI instead of URL here?

Reply:
It's also correct to say URI here, but I think URL is also correct and more concrete?

https://en.wikipedia.org/wiki/URL (see the Syntax section that gives `file://` as an example)

Reply:
gotcha, ok no problem then